recipes/dtw/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "dtw" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6be2d561cbebede2550d877b32ea4b5c058dced6a846603d7881d214c29f88bc
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --ignore-installed
+  number: 0
+
+requirements:
+  host:
+    - python >=3.4
+    - pip
+  run:
+    - python >=3.4
+    - numpy
+    - scipy
+
+test:
+  imports:
+    - dtw
+
+about:
+  home: https://github.com/pollen-robotics/dtw/
+  summary: 'Dynamic time warping is used as a similarity measured between temporal sequences'
+  description: |
+    Dynamic time warping is used as a similarity measured between temporal sequences.
+    Not to be confused with dtw-python (https://dynamictimewarping.github.io/).
+  license: BSD-3-Clause
+  license_family: GPL
+  license_file: LICENSE
```

Comment:
Maybe create an issue upstream that they should distribute the license file inside the pypi package as well.

Reply:
They do. So we do not need it here?